Code/Framework/AzFramework/AzFramework/Viewport/CameraInput.cpp

```diff
@@ -147,15 +159,18 @@ namespace AzFramework
             m_scrollDelta = scroll->m_delta;
         }
 
-        m_handlingEvents = m_cameras.HandleEvents(event, m_motionDelta, m_scrollDelta);
+        m_handlingEvents =
+            m_cameras.HandleEvents(event, ed_cameraSystemUseCursor ? m_cursorState.CursorDelta() : m_motionDelta, m_scrollDelta);
```

Comment:
Are you noticing a significant difference between the two options?

Reply:
Yeah it's massive unfortunately 🙁 I still think there's actually a bug lurking somewhere that we should be able to fix... as _in theory_ the behavior should be the same! But when using the editor (if you mouse over something, right click and wave the mouse around for a while, and then move it back to where you started, with the CursorDelta there's essentially no perceivable drift (checking in pitch/yaw of Go To Position will be 0, 0) but with the `motionDelta` way there's usually drift in pitch and yaw...) I haven't ripped the code out as I would like to investigate further at some point, but I've decided this is a good placeholder for now (and can be tested by flipping the CVar at runtime).

Reply:
No worries, if it works it works:) I guess it's now also consistent with the old editor viewport which we discovered used the delta between the cursor position instead of the raw mouse movement values like I originally thought. Would be interesting to know why it behaves so differently, but I wouldn't spend too much time on it if this gives us what we need.

Reply:
I've got a good test case locally now (basically we can accumulate both the `cursorState.CursorDelta()`s and `motionDeltas` in two separate accumulators, and then break when they first diverge).

There does seem to be some issues around conversions between floating point and integer values - something I noticed is you might sometimes have a value like `6.999999f` which when doing the standard `aznumeric_cast` to `int` will return `6` as all the extra precision is just thrown away. I had some success with using `AZStd::round` which will return `7` in this case but I still think there's an issue somewhere.

I'm going to fix some more pressing issues but I would like to circle back to this again as it really feels like there's something I don't fully understand (and really want to! 😝)